### PR TITLE
Test cli run

### PR DESCRIPTION
**Pull request by MarcinMoskala**

### CODE REVIEW

File: test/test_cli.py [hunk 1]
```diff
@@ -136,3 +139,66 @@ def test_should_find_root_package_when_root_used_project_name_used(self):
 
         # then
         self.assertEqual(test_module_src, resolve(res))
+
+    def test_should_run_workflow(self):
```
Comment (beczkowb) [inline test/test_cli.py:143]: When I read this test I wonder what's the difference between each workflow you run. Do they represent different situations?

File: test/test_cli.py [hunk 2]
```diff
@@ -136,3 +139,66 @@ def test_should_find_root_package_when_root_used_project_name_used(self):
...
+        # then
+        self.assert_started_jobs(['J_ID_3', 'J_ID_4'])
+
+        # when
+        cli_run(root_package, workflow_id="ID_4")
+
+        # then
+        self.assert_started_jobs(['J_ID_3', 'J_ID_4', 'J_ID_5'])
+
+    def test_should_run_workflow_multiple_times(self):
```
Comment (beczkowb) [inline test/test_cli.py:159]: Do we need a separate test for running multiple workflows?

Comment (MarcinMoskala) [reply test/test_cli.py:159]: No, but I separated it to have more smaller tests instead of one big

File: test/test_cli.py (hunk 1 above)
Comment (MarcinMoskala) [reply test/test_cli.py:143]: No, just different jobs and workflows. I check if when I run a workflow only this workflow is started, and when I start another one that only this other one is started.

**Pull request merged**